danger-kotlin-library/build.gradle

```diff
@@ -1,5 +1,5 @@
 plugins {
-    id 'org.jetbrains.kotlin.jvm' version '1.3.0'
+    id 'org.jetbrains.kotlin.jvm'
```

Comment:
No fixed version?

Reply:
not needed.. is defined in gradle properties